src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
 // write your code here
+const tr = document.querySelectorAll('tr');
+
+for (const item of tr) {
+  let move = document.createElement('item.tagName');
+
+  move = item.children[1].cloneNode(true);
+  item.lastElementChild.insertAdjacentElement('beforebegin', move);
```

Comment:
The use of `insertAdjacentElement('beforebegin', move)` is correct if you want to insert the cloned element before the last child of the `item`. Ensure that `move` is correctly defined before this operation.